src/main/java/uk/gov/justice/hmpps/casenotes/services/CaseNoteService.java

```diff
@@ -389,13 +389,16 @@ public int deleteCaseNotesForOffender(final String offenderIdentifier) {
     }
 
     @Transactional
-    @PreAuthorize("hasAnyRole('ROLE_DELETE_SENSITIVE_CASE_NOTES')")
-    public void softDeleteCaseNote(final String offenderIdentifier, final UUID caseNoteId) {
-        final var caseNote = repository.findById(caseNoteId).orElseThrow(() -> new EntityNotFoundException("Case note not found"));
+    @PreAuthorize("hasAnyRole('DELETE_SENSITIVE_CASE_NOTES')")
+    public void softDeleteCaseNote(final String offenderIdentifier, final String caseNoteId) {
```

Comment:
I am kinda curious on the naming.  Soft delete is really implementation detail is is not?